eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -102,7 +114,18 @@ function GetAdjustedReadmeContent($ReadmeContent, $PackageInfo, $PackageMetadata
   if ($codeOwnerArray) {
     Write-Host "Code Owners are $($codeOwnerArray -join ",")"
     $author = $codeOwnerArray[0]
-    $msauthor = $author # This is a placeholder for now. Will change to the right ms alias.
+    try {
+      $msauthor = ."$PSScriptRoot/Get-AADIdentityFromGithubUser.ps1" `
```

Comment:
Does this capture the output that was sent to write-host?

Reply:
Why does this have a `.` in front?

Reply:
We have return in script. We still print Write-Host to console, but return value to variable.